tests/installation/installation_settings/validate_ssh_service_enabled.pm

```diff
@@ -14,6 +14,7 @@ use Test::Assert 'assert_true';
 
 sub run {
     my $installation_settings = $testapi::distri->get_installation_settings();
+    $installation_settings->wait_for_overview_content_to_be_loaded();
```

Comment:
what about to run this method after the action that we know cause a refreshment, so basically we could do it internally after changing the service or the port, the test doesn't know about refreshments let's say, but we can go with this if you want it is a good workaround without having the solution in server side, wdyt?

Reply:
But in this case we don't do changes that cause the refreshment on Installation Setting. It's just initial page load when navigating from previous screen to Installation Settings.

Reply:
any reason why not to put it in `get_overview_content` ? At least I would have put it there :) but yeah, just to gathering feedback from you as I'm not sure about this and I had the same situation in my previous PRs. We can merge either way.

Reply:
The reason is that if we will need to create a test that checks, that the page has placeholders when it is loading, we'll not be able to do so. For sure that may never be the case, but we'll keep single responsibility principle which is also keep our solution more consistent.

Reply:
ahm I was referring to `is_shown()` which means that if the screen is loaded, I guess that method should be more complicated in this particular case, so we can break the single principle because we don't have the proper server solution. For the rest of screen we don't need to do that because there is not refreshment. With this approach we always have to add this new method to test (which we are not supposed to change much). Does it convince you? :)